### PR TITLE
[LM-10081] Overflow Menu

### PR DESCRIPTION
**Pull request by AnujLM**
## Description

**Features**

**Bugs**

-  Fixed Overflow Popup Menu

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Hotfix (fixes a breaking changes)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: lib/src/models/menu_decoration_model.dart [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+import 'package:flutter/material.dart';
+
+class LMMenuDecoration {
```
Comment (TyagiLikeMinds) [inline lib/src/models/menu_decoration_model.dart:4]: Can we use `PopUpMenuThemeData` class provided by Flutter in itself

File: lib/src/widgets/common/menu/menu.dart [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            Rect.fromPoints(
+              details.globalPosition,
+              details.globalPosition.translate(0, 0),
+            ),
+            overlay.localToGlobal(Offset.zero) & overlay.size,
+          );
+          showMenu(
+              context: context,
+              position: position,
+              elevation: menuDecoration?.elevation,
```
Comment (TyagiLikeMinds) [inline lib/src/widgets/common/menu/menu.dart:33]: Can we replace LMMenuDecoration class with PopUpMenuThemeData class provided by Flutter in itself

File: lib/src/widgets/conversation/chat_bubble.dart [hunk 3]
```diff
@@ -57,10 +54,8 @@ class LMChatBubble extends StatefulWidget {
   final LMTextView? outsideTitle;
   final Widget? outsideFooter;
   final Widget? mediaWidget;
-
-  final CustomPopupMenuController? menuController;
-  final Widget? menu;
   final List<LMMenuItemUI>? menuItems;
+  final LMMenuDecoration? menuDecoration;
```
Comment (TyagiLikeMinds) [inline lib/src/widgets/conversation/chat_bubble.dart:58]: `PopUpMenuThemeData` class

**Review by TyagiLikeMinds — CHANGES_REQUESTED** [review]

**Review by divyanshgandhilm — APPROVED** [review]

**Comment by divyanshgandhilm** [comment]
Behtereen! Now it looks good.

**Comment by divyanshgandhilm** [comment]
**CHANGELOG mentions**

Breaking changes in LMChatBubble - removed menu, and menuItems keys. Add deprecated tag in them.
Plus define the new variables - menuBuilder.

**Pull request merged**